src/Azure/Orleans.Journaling.AzureStorage/AzureAppendBlobStateMachineStorageOptions.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage;
+using Azure.Core;
+using Orleans.Runtime;
+
+namespace Orleans.Journaling;
+
+/// <summary>
+/// Options for configuring the Azure Append Blob state machine storage provider.
+/// </summary>
+public sealed class AzureAppendBlobStateMachineStorageOptions
+{
+    private BlobServiceClient? _blobServiceClient;
+
+    /// <summary>
+    /// Container name where state machine state is stored.
+    /// </summary>
+    public string ContainerName { get; set; } = DEFAULT_CONTAINER_NAME;
+    public const string DEFAULT_CONTAINER_NAME = "state";
```

Comment:
Does this container name have to be globally unique for the storage account? Should we prepend something here to avoid collisions?

Reply:
If a collision error will be obvious, I guess it's fine to leave as is.